tools/Google.Cloud.Tools.ReleaseManager.Tests/CheckVersionCompatibilityCommandTest.cs

```diff
@@ -0,0 +1,175 @@
+            new TestCase("1.0.0-beta01"),
+            // Straight to 1.0.0 is pretty bold, but if we don't have any previous releases, we don't have anything to compare it to.
+            new TestCase("1.0.0"),
+            // Beta to first stable release
+            // We perform the comparison with the beta even though we allow anything to change. It's still useful to see those changes.
+            new TestCase("1.0.0", "1.0.0-beta01") { { "1.0.0-beta01", Level.Major } },
+
+            // First beta of a new minor
+            new TestCase("1.1.0-beta01", "1.0.0-beta01", "1.0.0") { { "1.0.0", Level.Minor } },
+
+            // Second beta of a new minor
+            new TestCase("1.1.0-beta02", "1.0.0-beta01", "1.0.0", "1.1.0-beta01")
+            {
+                { "1.0.0", Level.Minor },       // We're aiming for 1.1.0, and there can't be breaking changes there
+                { "1.1.0-beta01", Level.Major } // We can break beta-only features, but we'd like to know that.
+            },
+
+            // Release of a new minor
+            new TestCase("1.1.0", "1.0.0-beta01", "1.0.0", "1.1.0-beta01")
+            {
+                { "1.0.0", Level.Minor },       // We're aiming for 1.1.0, and there can't be breaking changes there
+                { "1.1.0-beta01", Level.Major } // We can break beta-only features, but we'd like to know that.
+            },
+
+            // First beta of a new major
+            new TestCase("2.0.0-beta01", "1.0.0", "1.1.0") // Any beta releases in 1.x are irrelevant now
+            {
+                { "1.1.0", Level.Major }
+            },
+
+            // Second beta of a new major
+            new TestCase("2.0.0-beta02", "1.0.0-beta01", "1.0.0", "1.1.0-beta01", "1.1.0", "2.0.0-beta01")
+            {
+                { "1.1.0", Level.Major },        // So we know what we've broken compared with last GA
+                { "2.0.0-beta01", Level.Major }, // So we know what we've broken compared with last beta
+            },
+
+            // First stable release of a new major after a beta
+            new TestCase("2.0.0", "1.0.0-beta01", "1.0.0", "1.1.0-beta01", "1.1.0", "2.0.0-beta01")
+            {
+                { "1.1.0", Level.Major },        // So we know what we've broken compared with last GA
+                { "2.0.0-beta01", Level.Major }, // So we know what we've broken compared with last beta
+            },
+
+            // New major with no betas
+            new TestCase("2.0.0", "1.0.0-beta01", "1.0.0", "1.1.0-beta01", "1.1.0")
+            {
+                { "1.1.0", Level.Major },        // So we know what we've broken compared with last GA
+            },
+
+            // New patch (no prerelease)
+            new TestCase("1.1.1", "1.0.0", "1.1.0")
+            {
+                { "1.1.0", Level.Identical }
+            },
+
+            // New patch (with prerelease)
+            new TestCase("1.1.1-beta01", "1.0.0", "1.1.0")
```

Comment:
Can you add one with 2 patch betas so we can break on the second with respect to the first?

Reply:
Yup! Adding now.